### DISCUSSION TITLE
wireguard update not working, suggestions?

### DISCUSSION THREAD

**Discussion by nmbrg** [tteck/Proxmox#2943] [Q&A] [answered]
when i write 'update' in the console for the wireguard LXC, and after choosing to update the Wireguard LXC, it justs spins the green(?) line and the console line pops up in front the "updating lxc" text, and nothing happens. ive tested several times and rebooted the LXC several times, still no luck.

watch the attached video if my bad explanation isnt enough to understand what i mean.

https://github.com/tteck/Proxmox/assets/31845361/a2b1cbe3-aa69-4b68-8918-c10678c45e45

**Comment by tteck** [comment]
Wireguard updates via `apt update && apt upgrade` 

https://github.com/tteck/Proxmox/blob/0e01ceb460c72f096402c4f44e67471de6ef6628/ct/wireguard.sh#L63-L66

**Reply by nmbrg** [reply]
What is the update command for, that you mention in the page for wireguard? 

![IMG_2846](https://github.com/tteck/Proxmox/assets/31845361/d0f46b4d-36d3-4636-90ed-67c7fca0a7e4)

**Reply by kuchar91** [reply]
to install WGDashboard

**Reply by nmbrg** [reply]
Then why you say “To Update WireGuard or Install a
Dashboard, run the command below (or type update) in the LXC Console.” 

it clearly states that update is to be used if you want to update wireguard OR install dashboard. 

So, why you are giving mixed information on what are to be used?

heck, you are even given the choice between update LXC and installing dashboard when using ‘update’ in console.

so how are we supposed to know to use apt update && apt upgrade to update wireguard when you don’t even mention that in the info
And instead mention to use update?

**Reply by tteck** [reply, marked as answer]
When you use the command `update` and select "1" it executes `apt-get update && apt-get -y upgrade`. Look at the code!
https://github.com/tteck/Proxmox/blob/0e01ceb460c72f096402c4f44e67471de6ef6628/ct/wireguard.sh#L57-L68
